infra/base-images/base-builder/Dockerfile

```diff
@@ -138,6 +140,10 @@ RUN cd $SRC && \
 COPY precompile_afl /usr/local/bin/
 RUN precompile_afl
 
+RUN git clone https://github.com/ossf/fuzz-introspector.git fuzz-introspector && \
+    cd fuzz-introspector && \
+    git checkout 490fa0adba9f5d78ae3ff9df1cf68c5aac30f09e
```

Comment:
Since we're using LLVM 14 when cloning LLVM I would probably take a later commit from `fuzz-introspector`. Preferably most recent as of this writing which includes the LLVM upgrade in fuzz-introspector https://github.com/ossf/fuzz-introspector/commit/c27b4077f43424622d97c9b26c8f3cf779976070 and also some bug fixes.

Reply:
thanks for comments @DavidKorczynski